metricbeat/mb/event.go

```diff
@@ -109,27 +109,26 @@ func (e *Event) BeatEvent(module, metricSet string, modifiers ...EventModifier)
 //       "rtt": 115
 //     }
 func AddMetricSetInfo(module, metricset string, event *Event) {
-	info := common.MapStr{
-		"name":   metricset,
-		"module": module,
+	e := common.MapStr{
+		"event": common.MapStr{
+			"dataset": metricset,
+			"module":  module,
+		},
 	}
 	if event.Host != "" {
-		info["host"] = event.Host
+		e.Put("service.hostname", event.Host)
 	}
 	if event.Took > 0 {
-		info["rtt"] = event.Took / time.Microsecond
+		e.Put("event.duration", event.Took/time.Microsecond)
```

Comment:
This is described as being in nanoseconds in ECS.

Reply:
@spinscale What will happen if we send nano seconds data to ES versions prior 7?

Reply:
Good point, will convert it.

Also not related to nano second support in ES as it's not a timestamp.

Reply:
@andrewkroh As this is a breaking change, I wonder if we want to keep the old field around?

Other option would be to introduce the new field in nano seconds already in 6.6, deprecate the other one and then remove it in 7.0?

Reply:
The plan you describe sounds good to me. Have both in 6.6 and remove rtt in 7.0.